components/eventMap/EventMarker.tsx

```diff
@@ -0,0 +1,81 @@
+import { Ionicon } from "@components/common/Icons"
+import React from "react"
+import { ImageBackground, StyleSheet, Text, View } from "react-native"
+
+const MARKER_SIZE = 60
+
+export type EventMarkerProps = {
+  color: string
+  attendeeCount: number
+}
+
+export const EventMarker = ({ color, attendeeCount }: EventMarkerProps) => {
```

Comment:
```suggestion
export const EventMarkerView = ({ color, attendeeCount }: EventMarkerProps) => {
```
This is because we already have an `EventMarker` type below, and I don't want it to conflict with this.